src/migration_utilities.js

```diff
@@ -9,10 +9,33 @@ const argv = require("yargs")
   .version(false)
   .argv
 
+
+function getNetworkId(name) {
+  const { NETWORK_IDS } = require("@gnosis.pm/util-contracts/src/util/networkUtils")
+  return NETWORK_IDS[name.replace("-fork", "")].network_id
+}
+
+function getDependency(artifacts, network, deployer, path) {
```

Comment:
Did you know that network can be found on the deployer object?

Reply:
How the heck do you know this? 🗡

Reply:
I stumbled upon the objects that truffle injects the other day when working with @bh2smith on the snapshot plugin (since we had to create a web3 provider out of thin air).

Today, I just `console.log(deployer)` and saw all of this.

Reply:
You don't even have to pass the network object around, could just take it directly from the deployer, but whatever.

Reply:
> since we had to create a web3 provider out of thin air

This reminds me that we had an idea for a contribution to truffle that day... I have forgotten exactly what though.

Reply:
> This reminds me that we had an idea for a contribution to truffle that day... I have forgotten exactly what though.

We wanted to inject a web3 provider into the plugin methods. I spoke to them about it at TruffleCon and it seems like they intentionally didn't want to inject it there but only in the `truffle exec` scripts (their argument for why wasn't very clear though).